internal/civisibility/integrations/gotesting/reflections_test.go

```diff
@@ -0,0 +1,150 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package gotesting
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGetFieldPointerFrom tests the getFieldPointerFrom function.
+func TestGetFieldPointerFrom(t *testing.T) {
+	// Create a mock struct with a private field
+	mockStruct := struct {
+		privateField string
+	}{
+		privateField: "testValue",
+	}
+
+	// Attempt to get a pointer to the private field
+	ptr, err := getFieldPointerFrom(&mockStruct, "privateField")
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	if ptr == nil {
+		t.Fatal("Expected a valid pointer, got nil")
+	}
+
+	// Dereference the pointer to get the actual value
+	actualValue := (*string)(ptr)
+	if *actualValue != mockStruct.privateField {
+		t.Fatalf("Expected 'testValue', got %s", *actualValue)
+	}
+
+	// Modify the value through the pointer
+	*actualValue = "modified value"
+	if *actualValue != mockStruct.privateField {
+		t.Fatalf("Expected 'testValue', got %s", *actualValue)
```

Comment:
Should this be `Expected 'modified value', got %s`?

Reply:
you're totally right, I fixed that in [4b3cda6](https://github.com/DataDog/dd-trace-go/pull/2742/commits/4b3cda60a0994af37829bea5fa627dc0ac613566)